service/internal/zpages/templates/properties_table.html

```diff
@@ -1,4 +1,4 @@
-<b>{{.Name}}:</b>
+<h6>{{.Name}}</h6>
```

Comment:
Intentional change? Looks unrelated.

Reply:
Was related to when I looked at the zpages :) Will split in a separate PR.